Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs

```diff
@@ -10,5 +10,8 @@ public class ItSystemUsageOverviewReadModel : IOwnedByOrganization, IReadModel<I
         public string Name { get; set; }
         public bool ItSystemDisabled { get; set; }
         public bool IsActive { get; set; }
+        public string ItSystemParentName { get; set; }
+        public string Version { get; set; }
+        public string LocalCallName { get; set; }
```

Comment:
Tilføj også ParentItSystemId som en nullable int (int?)

Reply:
På den måde kan jeg lave et aktivt link i frontend i stedet for de der døde tekster